torch/csrc/multiprocessing/init.cpp

```diff
@@ -38,6 +39,15 @@ PyObject* multiprocessing_init(PyObject* _unused, PyObject* noargs) {
   Py_RETURN_TRUE;
 }
 
+PyObject* set_thread_name(PyObject* _unused, PyObject* arg) {
```

Comment:
Maybe make a `get_thread_name` method too - just in-case someone wants it?

Reply:
done :)